src/main/java/kevesse_kokanyolo_kod/effects/CleaningEffect.java

```diff
@@ -0,0 +1,32 @@
+package kevesse_kokanyolo_kod.effects;
+
+import kevesse_kokanyolo_kod.items.Item;
+import kevesse_kokanyolo_kod.player.Professor;
+import kevesse_kokanyolo_kod.player.Student;
+
+public class CleaningEffect extends RoomEffect{
+    private int visitsBeforeEffect;
+
+    protected CleaningEffect(Item givenBy, int duration, EffectConsumedObserver observer) {
+        super(null, 0, null);
```

Comment:
Ez azért null, 0, null mert rögtön törlődik?
És ha igen, akkor miért létezik az osztály miért nem egy clean() metódust kap a Room?

Reply:
Mivel ez is logikailag egy effektje a szobának, ezért azt gondoltam ennek is itt van a helye, és mivel időben mérendő hatása és emiatt lejárta sincs az osztálynak amit obserrverrel kéne nézni emiatt nullok azok az értékek